<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Room_cal_total_widget</class>
 <widget class="QWidget" name="Room_cal_total_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1203</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#room_cal_total
{
	background-color: rgb(255, 255, 255);
	border:1px solid black;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget_info" native="true">
     <property name="minimumSize">
      <size>
       <width>1185</width>
       <height>82</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1185</width>
       <height>161</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#widget_info
{
	background-color: rgb(255, 255, 255);
	border:1px solid black;
}

QLabel
{
font-family: '黑体';
font-size: 16px;
}

QLineEdit
{
font-family: '黑体';
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="text">
         <string>主竖区 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_MVZName">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="text">
         <string>甲板 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_deck">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_cg1">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="text">
         <string>房间编号 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_room_number">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="text">
         <string>噪声限值 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_noise_limit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_cg2">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="text">
         <string>主风管数量 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_duct_num">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="minimumSize">
          <size>
           <width>661</width>
           <height>431</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">/*设置标题(包括垂直+水平的)*/
QHeaderView
{							
	
	font-size: 15px;				/*字大小*/
	font-family: '黑体';
	/*border: 1px solid #B4B4B4;*/
	border:none;
    /*border-bottom: 2px solid #999999; */		/*下边框颜色*/
    /*background: #9DC6E6;*/				/*背景浅蓝色*/ 
	min-height:30px;
	text-align: center;	       
}


/*设置标题(水平的)*/
QHeaderView::section:horizontal { 					
        border: 1px solid #AFAFAF; 			/*间隔颜色*/
	    border-bottom: 2px solid #999999; 							
        color: black;
        background: #9DC6E6;
}

/*设置鼠标停留状态*/
QHeaderView::section:horizontal:hover 
{			
        color: black;							/*字体白色*/
        background: #82AFC9;				/*背景深蓝色*/
}

/*设置鼠标按下状态*/
QHeaderView::section:horizontal:pressed 
{			
        color: black;
        background: #82AFC9;
		border-bottom: 3px solid #037BFC;				
}


QTableWidget 
{
		font-size: 14px;				/*10榜*/
		font-family: '黑体';				
		color : black;				
        border: 1px solid #000000;		
        background: white;
        gridline-color: #E0E0E0;	
        text-align: center;	
        outline:0px;			/*禁止焦点*/
}

/*设置选中的视图项*/
QTableWidget::item:selected
{				    
		border: 2px solid #0398FC;	
        color : black;				
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>主风管编号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>测试点距离(m)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>A级计权噪音dB(A)</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>661</width>
           <height>41</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>661</width>
           <height>41</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,7">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_cg3">
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="text">
             <string>房间噪音总值dB(A):</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_totalNoise">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 1px solid black ;
border-left:none;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
